EDMarketConnector.py

```diff
@@ -806,30 +870,38 @@ def cooldown(self):
                                                                  monitor.system and
                                                                  tk.NORMAL or tk.DISABLED)
 
-    def ontop_changed(self, event=None):
+    def ontop_changed(self, event=None) -> None:
+        """Set main window 'on top' state as appropriate."""
         config.set('always_ontop', self.always_ontop.get())
         self.w.wm_attributes('-topmost', self.always_ontop.get())
 
-    def copy(self, event=None):
+    def copy(self, event=None) -> None:
+        """Copy system, and possible station, name to clipboard."""
         if monitor.system:
             self.w.clipboard_clear()
             self.w.clipboard_append(monitor.station and f'{monitor.system},{monitor.station}' or monitor.system)
 
-    def help_general(self, event=None):
+    def help_general(self, event=None) -> None:
+        """Open Wiki Help page in browser."""
         webbrowser.open('https://github.com/EDCD/EDMarketConnector/wiki')
 
-    def help_privacy(self, event=None):
+    def help_privacy(self, event=None) -> None:
+        """Open Wiki Privacy page in browser."""
         webbrowser.open('https://github.com/EDCD/EDMarketConnector/wiki/Privacy-Policy')
 
-    def help_releases(self, event=None):
+    def help_releases(self, event=None) -> None:
+        """Open Releases page in browser."""
         webbrowser.open('https://github.com/EDCD/EDMarketConnector/releases')
 
     class HelpAbout(tk.Toplevel):
+        """The applications Help > About popup."""
+
```

Comment:
extra newline?

Reply:
Newline after *class* docstring is mandatory.

Annoyingly *not* having such after function docstring is mandatory.  It keeps tripping me up :P .

Reply:
Yeah it likes catching me too